content/hi/docs/tutorials/kubernetes-basics/scale/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: अपने ऐप को स्केल करें 
```

Comment:
Can you localise "Application scaling" to Hindi?

Reply:
`अपने ऐप को स्केल करें` seems the good translation for `Scale Your App`.